samples/unit-tests/series-item/dynamic/demo.js

```diff
@@ -39,3 +39,16 @@ QUnit.test('Item series dynamics', assert => {
         'Point length modified'
     );
 });
+
+QUnit.test('Errors in the console during initialization with no data should not be visible, #13379.', assert => {
```

Comment:
This test is good, but I'm thinking we should take this opportunity to go further, and make a general unit test so that we don't do this error again.

We should have a generic test that checks that all series types behave nicely with 
- missing `series` option
- `series: []`
- missing `data` option
- `data: []`

See https://github.com/highcharts/highcharts/blob/v8.1.0/samples/unit-tests/styled-mode/series-types/demo.js#L45-L121 for inspiration. We can create this test in `unit-tests/series/seriestypes`